### ISSUE TITLE
No output 

### ISSUE THREAD

**Issue by dualfade** [boringthegod/CloudFade#1]
Good morning. I was testing this out this past weekend. I would like to add it to the blackarch repo's, This really looks great. However I am not getting any actual output ? or anything written to a log file. While monitoring dnscrypt logs I can see the umask work, but only from the local system perspective of 127.0.0.1. 

I am on Archlinux latest blackarch package strapped.

Thanks in advance !

**Comment by boringthegod** [comment]
Can you please send me the command you typed so that I can reproduce it?

**Comment by dualfade** [comment]
Sure, 
Here ya go --

```zsh
/tmp % uname -a
Linux magnetar 6.12.4-arch1-1 #1 SMP PREEMPT_DYNAMIC Mon, 09 Dec 2024 14:31:57 +0000 x86_64 GNU/Linux
/tmp %
```

```zsh
CloudFade[master] % which cloud_fade
/home/dualfade/bin/cloud_fade
CloudFade[master] % cloud_fade --version
CloudFade 1.4

CloudFade[master] % cloud_fade --aggressive -d stena.com --iprange 194.103.125.0/24 --useragents /home/dualfade/Weapons/wordlists/SecLists/Fuzzing/User-Agents/user-agents-whatismybrowserdotcom-small.txt --output /tmp/cf_stena.txt
Use of : 320 threads
  [00:00:50] [########################################] 254/254 (0s)
CloudFade[master] % b /tmp/cf_stena.txt
CloudFade[master] % ll /tmp/cf_stena.txt
-rw-r--r-- 1 dualfade dualfade 0 Dec 18 10:00 /tmp/cf_stena.txt
CloudFade[master] %

```

however I can see it unmasked in my dnscrypt logs --

```zsh
[2024-12-18 10:02:37]   127.0.0.1       whois.ripe.net  AAAA    PASS    0ms     -
[2024-12-18 10:02:37]   127.0.0.1       whois.ripe.net  AAAA    PASS    0ms     -
[2024-12-18 10:02:37]   127.0.0.1       whois.ripe.net  A       PASS    0ms     -
[2024-12-18 10:02:38]   127.0.0.1       stena.com       AAAA    PASS    0ms     -
[2024-12-18 10:02:38]   127.0.0.1       stena.com       A       PASS    0ms     -
[2024-12-18 10:02:38]   127.0.0.1       stena.com       AAAA    PASS    0ms     -
[2024-12-18 10:02:38]   127.0.0.1       stena.com       A       PASS    0ms     -
[2024-12-18 10:02:39]   127.0.0.1       stena.com       AAAA    PASS    0ms     -
[2024-12-18 10:02:39]   127.0.0.1       stena.com       A       PASS    0ms     -
[2024-12-18 10:02:39]   127.0.0.1       stena.com       AAAA    PASS    0ms     -
[2024-12-18 10:02:39]   127.0.0.1       stena.com       A       PASS    0ms     -
```

Let me know if you need anything else ? I am running the binary release as I cannot get the git version to compile without error. Here is that error if you would like it ?

```zsh
CloudFade[master] % cargo build --release
   Compiling rquest v0.27.7
error[E0599]: no method named `reset_pool_idle` found for struct `hyper2::Client<Connector, ImplStream>` in the current scope
    --> /home/dualfade/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rquest-0.27.7/src/client/http.rs:1551:26
     |
1551 |         self.inner.hyper.reset_pool_idle();
     |                          ^^^^^^^^^^^^^^^ method not found in `Client<Connector, ImplStream>`

error[E0599]: no method named `reset_pool_idle` found for struct `hyper2::Client<Connector, ImplStream>` in the current scope
    --> /home/dualfade/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rquest-0.27.7/src/client/http.rs:1567:26
     |
1567 |         self.inner.hyper.reset_pool_idle();
     |                          ^^^^^^^^^^^^^^^ method not found in `Client<Connector, ImplStream>`

error[E0599]: no method named `reset_pool_idle` found for struct `hyper2::Client<Connector, ImplStream>` in the current scope
    --> /home/dualfade/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rquest-0.27.7/src/client/http.rs:1577:26
     |
1577 |         self.inner.hyper.reset_pool_idle();
     |                          ^^^^^^^^^^^^^^^ method not found in `Client<Connector, ImplStream>`

error[E0599]: no method named `reset_pool_idle` found for struct `hyper2::Client<Connector, ImplStream>` in the current scope
    --> /home/dualfade/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rquest-0.27.7/src/client/http.rs:1587:26
     |
1587 |         self.inner.hyper.reset_pool_idle();
     |                          ^^^^^^^^^^^^^^^ method not found in `Client<Connector, ImplStream>`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `rquest` (lib) due to 4 previous errors
CloudFade[master] %

```

Thanks  !

**Comment by boringthegod** [comment]
![image](https://github.com/user-attachments/assets/6d75b3ba-b741-4da9-a0c3-f2ce81349c3d) 

Isn't it just because there's no IP that lets you hit the site without going through CloudFare?


Try : `cloud_fade --domain www.monplancbd.fr --iprange 51.15.10.120-51.15.10.122`

There should be an IP found to type the site directly

**Comment by dualfade** [comment]
Huh interesting. Maybe you should have an example on the page of what good output looks like ? 

```zsh
python exploit.py --domain 'stena.com' --cidr 194.103.125.0/24 --threads 10 --outfile /tmp/outfile_$(date +%s).txt
ic => "[info] %s Starting domain unmask"
      % strftime("%Y-%m-%d %H:%M:%S =>", localtime()): '[info] 2024-12-19 07:56:32 => Starting domain unmask'
ic => "[info] %s Domain %s resolved to %s by DNS server %s"
      % (strftime("%Y-%m-%d %H:%M:%S =>", localtime()), domain, result, server): ('[info] 2024-12-19 07:57:01 => Domain stena.com resolved to 194.103.126.81 by '
                                                                                  'DNS server 194.103.125.36')
ic => "[info] %s Writing to file %s"
      % (strftime("%Y-%m-%d %H:%M:%S =>", localtime()), outfile): '[info] 2024-12-19 07:57:01 => Writing to file /tmp/outfile_1734612992.txt'

```
Then I can figure out which host is actually there. as an example. Let me try a few more things and ping you back. Appreciate it. 

Thanks  !

**Comment by dualfade** [comment]
Alright I guess I will close this. Must be my side. Much appreciated.
